src/views/confirmation/confirmation.view.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC, useEffect, useState } from "react";
+
+import { ReactComponent as ArrowRightIcon } from "src/assets/icons/arrow-right.svg";
+import useConfirmationStyles from "src/views/confirmation/confirmation.styles";
+import Header from "src/views/shared/header/header.view";
+import { useTransactionContext } from "src/contexts/transaction.context";
+import Card from "src/views/shared/card/card.view";
+import TokenIcon from "src/views/shared/token-icon/token-icon.view";
+import Typography from "src/views/shared/typography/typography.view";
+import { useNavigate } from "react-router-dom";
+import routes from "src/routes";
+import { tokenAmountToNumber, convertTokenAmountToFiat } from "src/utils/amounts";
+import Button from "src/views/shared/button/button.view";
+import Error from "src/views/shared/error/error.view";
+
+const Confirmation: FC = () => {
```

Comment:
I would rename this component `TransactionConfirmation` or `BridgeConfirmation`, as `Confirmation` looks very generic and reusable. Same for the `routes` path.